### ISSUE TITLE
Introduce a neutral type of change in changelog

### ISSUE THREAD

**Issue by adamkudrna** [react-ui-org/react-ui#418]
Something can be just a change — not a feature, not refactoring, not patch, just a change.

**Comment by mbohal** [comment]
I disagree, I think we should close this issue. Nobody does changes for no reason and therefore the current categories should be enough.

Generally we make changes with the aim to:
* improve something -> `feature`
* fix something -> `bugfix`
* improve code quality -> `refactoring`

I can not envision change that would not fall in these categories.

Also the more categories we have the more complicated will the contribution process get. And we do not want it to be complicated.

**Comment by adamkudrna** [comment]
What if you change e.g. `Tabs` border color to a slightly different shade of gray? It doesn't introduce anything new nor it doesn't fix anything. It may be a design improvement (although calling it a _feature_ sounds too much to me), but from the code point of view it's just a change.

**Comment by mbohal** [comment]
> It doesn't introduce anything new nor it doesn't fix anything. 

Why bother doing it then when it brings no benefit?

**Comment by adamkudrna** [comment]
A better design may be the benefit. For me, that is worth doing.

**Comment by mbohal** [comment]
> A better design may be the benefit. For me, that is worth doing.

But a design improvement is a feature, no? As you wrote above:

> improve something -> feature

If it is not a `feature`, I could envision a `design` type change. I think we don't need it, but if you think we do I'm fine with it.
